packages/worker/src/api/controllers/global/auth.js

```diff
@@ -145,6 +145,32 @@ exports.logout = async ctx => {
   ctx.body = { message: "User logged out." }
 }
 
+exports.datasourcePreAuth = async (ctx, next) => {
```

Comment:
Love that these endpoints are datasource agnostic 👍

Reply:
![](https://media.giphy.com/media/YOvOkaS5ZKfimDIgwJ/giphy.gif)